post-scan-actions/aws-python-slack-notification/handler.py

```diff
@@ -0,0 +1,45 @@
+import urllib3
+import json
+import os
+http = urllib3.PoolManager()
+from botocore.exceptions import ClientError
```

Comment:
Seem not used.

Reply:
The only one that I'm not using there is the from botocore.exceptions import ClientError